components/menu/index.css

```diff
@@ -545,8 +574,6 @@
 .spectrum-Menu-chevron {
 	grid-area: chevronArea;
 	align-self: center;
-	block-size: var(--spectrum-menu-item-checkmark-height);
-	inline-size: var(--spectrum-menu-item-checkmark-width);
```

Comment:
So were the `block-size` and `inline-size` actually styles for the icon (`spectrum-Icon`), which is why the grid area and align-self can be left separately?

Reply:
`block-size` and `inline-size` were just moved up to the existing chevron selector (see `.spectrum-Menu-chevron.spectrum-Icon`).

I left the grid and alignment where they were since all the  other grid and alignment styles are still there next each other, and I didn't want to get carried away with refactoring.